_posts/2015-09-23-rentrez-1-released.md

```diff
@@ -0,0 +1,236 @@
+---
+name: rentrez-1_0-release
+layout: post
+title: Rentrez 1.0 released 
+date: 2015-09-23
+authors: 
+  - name: David Winter
+  - url: http://cartwrig.ht/people/#david-j-winter
+tags:
+- R
+- CRAN
+- rentrez
+- packages
+- NCBI
+---
+
+
+A new version of `rentrez`, our package for the NCBI's EUtils API, is making
+it's way around the CRAN mirrors. This release represents a substantial
+improvement to `rentrez`, including a [new vignette](https://cran.r-project.org/web/packages/rentrez/vignettes/rentrez_tutorial.html)
+that documents the whole package. 
+
+This posts describes some of the new things in `rentrez`, and gives us a chance
+to thank some of the people that have contributed to this package's development. 
+
+## Thanks
+
+Thanks to everyone who has filed and issue or written us an email about `rentrez`, 
+your contributions have been an important part of the package's development. In particular, we welcome
+[Han Guangchun](http://ewrebio.me) as a new contributor to `rentrez` and thank 
+[Matthew O'Meara](http://docking.org/~momeara/) for posting
+an issue that brought the `by_id` mode for `entrez_link`  (discussed below) to our
+attention.
+
+##The New Stuff
+
+###Extract elements from the results of `entrez_summary()`
+
+The NCBI's "summary records" are very useful -- they provide the most important
+information about a given record in a relatively small and simple file. `rentrez`
+provides the function `entrez_summary()` to retrieve these records. When more
+than one unique ID is passed to `entrez_summary` the function returns a list of
+esummary objects. For instance, you could find all the genetic variants associated
+with asthma by finding links between the [OMIM record for asthma](http://www.omim.org/entry/600807) and records in the database dbSNP:
+
+
+```r
+snps <- entrez_link(dbfrom="omim", db="snp", id= 600807)
+snp_summs <- entrez_summary(db="snp", id=snps$links$omim_snp)
+```
+
+A very common use-case for `entrez_summary()` is to extract a subset of the
+elements from each record in that list. This release includes the function
+`extract_from_esummary` to make this as straightforward as possible. It works with
+a single element to extract:
+
+
+```r
+extract_from_esummary(snp_summs, "chr")
+```
+
+```
+## 11079657  2786098  1031772  1031771   545659 
+##     "17"      "1"      "2"      "2"     "11"
+```
+
+Or with multiple elements
+
+```r
+summary_table <- extract_from_esummary(snp_summs, c("chr", "fxn_class"))
+knitr::kable(t(summary_table))
+```
+
+|chr |fxn_class               |
```

Comment:
Not sure why but I think we have to make actual html tables to make this work, but after merging I can do that since it's a very small table